modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_request.rb

```diff
@@ -33,6 +33,8 @@ module ClaimantTypes
     before_validation :set_claimant_type
 
     validates :claimant_type, inclusion: { in: ClaimantTypes::ALL }
+    validates :power_of_attorney_holder_type, inclusion: { in: PowerOfAttorneyHolder::Types::ALL }
```

Comment:
@jvcAdHoc Another nice option here is that we've used elsewhere (forthcoming PRs still that have this pattern):
```ruby
enum(
  :status,
  PowerOfAttorneyHolder::Types::ALL.index_by(&:itself),
  validate: true
)
```